tests/cupyx_tests/scipy_tests/interpolate_tests/test_ppoly.py

```diff
@@ -22,44 +19,35 @@
 
 # TODO: add BPoly, if/when implemented
 interpolate_cls = ['PPoly']
-numpy_params = [(numpy, scipy)]
-cupy_params = [(cupy, cupyx.scipy)]
-
-numpy_params = product(numpy_params, interpolate_cls)
-cupy_params = product(cupy_params, interpolate_cls)
-params = chain(numpy_params, cupy_params)
-params = [(x, y, z) for ((x, y), z) in params]
-
-parametrize_cls_xp = pytest.mark.parametrize('xp,scp,cls', params)
-parametrize_cls = pytest.mark.parametrize('cls', interpolate_cls)
 
 
 @testing.with_requires("scipy")
 class TestPPolyCommon:
     """Test basic functionality for PPoly and BPoly."""
 
-    @testing.with_requires('scipy')
-    @parametrize_cls_xp
+    @pytest.mark.parametrize('cls', interpolate_cls)
```

Comment:
`xp` and `scp` needs to be parameterized. Same for other tests.

Reply:
Fixed!